packages/react-native-web/src/exports/Modal/ModalAnimation.js

```diff
@@ -32,7 +32,7 @@ export type ModalAnimationProps = {|
   visible?: ?boolean
 |};
 
-function ModalAnimation(props: ModalAnimationProps) {
+function ModalAnimation(props: ModalAnimationProps): any | null {
```

Comment:
Why `any | null`?

Reply:
`createElement` isn't typed so it's returning any

Reply:
(I'm going to try to throw a very loosey / any type on createElement)